applications/ContactStructuralMechanicsApplication/custom_strategies/custom_convergencecriterias/penalty_frictional_mortar_criteria.h

```diff
@@ -105,6 +105,9 @@ class PenaltyFrictionalMortarConvergenceCriteria
         const bool IODebug = false
         ) : BaseType(ComputeDynamicFactor, IODebug, PureSlip)
     {
+        // Initialize STD output
+        ColorUtilities::InitializeSTDOutput();
```

Comment:
didn't we say that this should be included in the logger?
Otherwise it is kinda unmaintainable